### DISCUSSION TITLE
How do vtracer compare with https://vectorizer.ai/

### DISCUSSION THREAD

**Discussion by 6174** [visioncortex/vtracer#42] [General]
I test both vectorizer.ai and vtracer with the image below, looks like the result vectorizer is way better than vtracer. How do you compare with vectorizer.ai?  
![image](https://user-images.githubusercontent.com/3872872/231651425-113a6e06-b120-41bd-bb5d-1963c47abc2c.png)

**Comment by tyt2y3** [comment]
Surely it does, because the image suffers from jpeg noise. We did not model that or take that into consideration.
I imagine a denoising pass would improve the result a lot.
But I cannot comment on proprietary technologies without a technical description.

**Reply by 6174** [reply]
Cool, looks like very promising.
